ecs-agent/tmds/handlers/fault/v1/handlers/handlers.go

```diff
@@ -214,6 +214,12 @@ func (h *FaultHandler) startNetworkBlackholePort(ctx context.Context, protocol,
 			})
 			return string(cmdOutput), err
 		}
+		logger.Info("[INFO] Successfully created new chain", logger.Fields{
```

Comment:
nit: we don't have to put the log level explicitly. the logger already takes care of it. ie the beginning of the log entry already has the level - 
`level=info time=2024-10-11T02:32:12Z msg="[INFO] Black hole port fault is not running" output="iptables: Bad rule (does a matching rule exist in that chain?).\n" taskArn="arn:aws:ecs:us-west-2:817190723229:task/harishxr-ecs/fa06536c65eb46af8f556fb42862b45b" exitCode=1 netns="host" command="iptables -w 5 -C egress-tcp-1234 -p tcp --dport 1234 -j DROP"`

Reply:
+1 we can remove those. forgot to cleaned those up after testing locally

Reply:
Address in [1e5cb61](https://github.com/aws/amazon-ecs-agent/commit/1e5cb61d395aeee30b0583c7cb0b30e339c6c4ef)